test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -74,7 +74,17 @@ elif [[ $FRAMEWORK == "mxnet" ]]; then
         exit $exit_code
     fi
 else
-    pip install tensorflow-datasets
+    if [[ $FRAMEWORK == "tensorflow2" ]]; then
```

Comment:
I will advise to just remove, all if else and pin the tf dataset to a working version.